app/src/main/res/values/strings.xml

```diff
@@ -137,4 +127,11 @@
     <string name="no_comrade_title">No Comrades Registered</string>
     <string name="no_comrade_msg">Message not send to any comrade, since none is registered</string>
 
+    <!-- INTRO screen strings-->
```

Comment:
You have misunderstood what I have said. Please keep them under the Circle of trust section. Remove the strings you added.